README.md

```diff
@@ -260,6 +260,41 @@ We have a small collection of algorithms, one for every day of the month. Scroll
 - **Uses and Resources**
   - This question is a brain teaser and though it may not seem to have any practical application, it is used in various areas such as Crypto-Analysis and Brute-forcing cryptographic keys.
 
+### **December 14 - Ceaser Cipher**
+- **Problem**
+  - There is a specific code in which after the encoding process is done each of the characters are moved by a certain position. The number of positions by which they are moved is given by the number of characters that are present in the given word. This problem requires you to take the input string and encrypt it according to this system and add an decrypting algorithm. In the case of the last few alphabets it should move back again to the first (z->a)
```

Comment:
The Caesar cipher is one of the earliest known ciphers. Simply put, it should take in an input of a 'key' and some plain text. This key will be a number from 0-25. Your task is to simply replace each character of the message with a new character. This new character is the key-th element from the old character. Check out the examples for better understanding.  Also, develop a function to decrypt the given cipher when you are given the key.

Optional: The Caesar cipher is trivial for modern computers to crack, even without the key. Write a program that bruteforces the cipher. Bruteforcing means simply trying out every possible combination. So in this case, try out every possible key to decode the cipher.